### PR TITLE
React i18next context exports

### PR DESCRIPTION
**Pull request by MichaReiser**
Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [context.js](https://github.com/i18next/react-i18next/blob/master/src/context.js)
- [x] Increase the version number in the header if appropriate. 
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. -> already exists

There are still some typings missing to fully cover react-i18next. I will add them when needed.

### CODE REVIEW

File: types/react-i18next/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-i18next 4.6
+// Type definitions for react-i18next 7.0.0
```
Comment (GiedriusGrabauskas) [inline types/react-i18next/index.d.ts]: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md

File: types/react-i18next/index.d.ts [hunk 2]
```diff
@@ -22,6 +22,9 @@ export {
     TranslationFunction
 };
 
+export { reactI18nextModule, setDefaults, getDefaults, setI18n, getI18n } from './src/context';
+
+
```
Comment (GiedriusGrabauskas) [inline types/react-i18next/index.d.ts]: Consecutive blank lines are forbidden.

**Review by GiedriusGrabauskas — CHANGES_REQUESTED** [review]
Please fix Travis.

**Comment by MichaReiser** [comment]
I do not see any specific reason why this change should break with Typescript@next. Might these be caused by an issue with typescript itself or does anyone of you an idea what is wrong with the typings?

**Comment by MichaReiser** [comment]
Closing in favour of #21881

**Pull request closed without merging**